models/sipnet/inst/register.SIPNET.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<SIPNET>
```

Comment:
I'm not a fan of the design decision to put the model name as the outer block in the XML. If you need to specify the model name anywhere it should be internally e.g. as ```<name>```